homeassistant/components/spotify/media_player.py

```diff
@@ -1,24 +1,22 @@
 """Support for interacting with Spotify Connect."""
 from __future__ import annotations
 
-from asyncio import run_coroutine_threadsafe
 import datetime as dt
-from datetime import timedelta
 import logging
+from asyncio import run_coroutine_threadsafe
+from datetime import timedelta
 from typing import Any
 
 import requests
-from spotipy import SpotifyException
 from yarl import URL
 
-from homeassistant.components.media_player import (
-    BrowseMedia,
-    MediaPlayerEntity,
-    MediaPlayerEntityFeature,
-    MediaPlayerState,
-    MediaType,
-    RepeatMode,
-)
+from homeassistant.components.media_player import (ATTR_MEDIA_ENQUEUE,
+                                                   BrowseMedia,
+                                                   MediaPlayerEnqueue,
+                                                   MediaPlayerEntity,
+                                                   MediaPlayerEntityFeature,
+                                                   MediaPlayerState, MediaType,
+                                                   RepeatMode)
```

Comment:
This change seems to be unrelated/wrong? Have you set up a dev env as per our developer documentation?

Reply:
My bad!